.gitignore

```diff
@@ -8,7 +8,6 @@
 .mypy_cache
 .hypothesis
 .tox
```

Comment:
```suggestion
.tox
.python-version
```

Reply:
Are we adding back `.python-version`?  I removed it because I deleted my local copy of `.python-version`.  It was causing issue in `zsh` when switching to `Python 3.6`.

Reply:
Yes. It's what pyenv users commonly use. I don't know what your problem was but it was probably specific to your (mis)configuration or maybe you just used it wrong. Anyway, even if you stop using pyenv, you shouldn't prevent the contributors from using it.